.github/workflows/release-post-merge.yml

```diff
@@ -12,14 +12,36 @@ on:
       version:
         description: "Release version (Be sure `Release-branch` is successful):"
         required: true
-
+      image_repo:
```

Comment:
Once approved, should we set the default to the one actually used when releasing?

Reply:
These defaults only apply for inputs, for prod it will fallback to fixed values, see env section.